### PR TITLE
Scheduled updates: Configure multisite routes

### PR DESCRIPTION
**Pull request by bogiii**
Related to https://github.com/Automattic/wp-calypso/issues/89473

## Proposed Changes

* Defines `index`, `create`, and `edit` routes for multisite context
* Introduces feature flag: `plugins/multisite-scheduled-updates`
* Renames module folder name

The idea is to keep the same entry route for this feature. On the other side, we will not see the site selector anymore since if you don't provide the siteSlug, it means multisite context.

## Testing Instructions

* Go to `/plugins/scheduled-updates` and check if there is a proper placeholder
* Go to `/plugins/scheduled-updates/create` and check if there is a proper placeholder
* Go to `/plugins/scheduled-updates/edit/random-string-as-schedule-id` and check if there is a proper placeholder
* Pass through the existing site-specific Scheduled Updates feature and check if everything works as before

Screenshot 2024-04-12 at 14 09 01
Screenshot 2024-04-12 at 14 09 17
Screenshot 2024-04-12 at 14 09 38

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/plugins/controller.js [hunk 1]
```diff
@@ -170,6 +170,24 @@ export function updatesManager( context, next ) {
 	next();
 }
 
+export function updatesManagerMultisite( context, next ) {
```
Comment (obenland) [inline client/my-sites/plugins/controller.js]:
```suggestion
export function scheduledUpdatesMultisite( context, next ) {
```

Let's maintain Scheduled Updates as much as possible as the feature name, so it's easier to identify across code bases

Comment (bogiii) [reply client/my-sites/plugins/controller.js]: Sure! Also, I'll rename the folder where the feature logic lives from "plugins-update-manager" to "plugins-scheduled-updates".

**Review by ouikhuan — APPROVED** [review]
Other than naming it works as expected 👍  However, we probably want to put a feature flag for multi-site context for now? At least for `/plugins/scheduled-updates`? It shows a site selector component at the moment.

**Comment by bogiii** [comment]
I've renamed the module folder and controller function to have "plugins-scheduled-updates".
Also, the feature flag `plugins/multisite-scheduled-updates` has been introduced, and without a flag, everything works as before this PR.

@ouikhuan, it's ready for review.

**Review by ouikhuan — APPROVED** [review]
LGTM 🚢

**Pull request merged**